src/module/rules/SoakRules.ts

```diff
@@ -29,6 +29,21 @@ export class SoakRules {
             return Helpers.reduceDamageByHits(damageData, damageData.value, 'SR5.VehicleStunImmunity');
         }
 
+        let armorData = actor.getArmor(damageData)
+
+        // @ts-expect-error
+        if(armorData.hardened) {
+            let ap = armorData.mod.filter(mod => mod.name === "SR5.AP")[0].value
+            let base = armorData.base
+            let modifiedArmor = base + ap;
+
+            if(damageData.value < modifiedArmor) {
+                hits += damageData.value
```

Comment:
Although this is an effective solution to negate all damage, I think a solution similar to `CombatRules:modifyDamageAfterMiss` might be more appropriate, overriding the damage to 0.

Reply:
What is the benefit?

Reply:
I don't think it affects the actual output, but I think setting the override value is more idiomatic and reflective of the actual rule being implemented.  One could read the previous implementation and be confused that there was a rule wherein the physical defense roll actually gained net hits in this condition.